src/javascript/JContent/ContentRoute/ContentLayout/Upload/UploadItem/EditButton.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {uploadStatuses} from '../Upload.constants';
+import * as _ from 'lodash';
+import {DisplayActions} from '@jahia/ui-extender';
+import {ButtonRendererNoLabel} from '~/utils/getButtonRenderer';
+import PropTypes from 'prop-types';
+import {isImageFile} from '../../ContentLayout.utils';
+
+const EditButton = props => {
+    const {status, file, path} = props;
+
+    if (window.contextJsParameters.config.links.tagImageOnUpload === 'true' && isImageFile(file.name)) {
+        if (status === uploadStatuses.UPLOADED) {
+            return (
+                <DisplayActions
+                    target="contentActions"
+                    filter={value => _.includes(['edit'], value.key)}
```

Comment:
I'm not sure why _.includes is needed for a straight up string comparison? isn't value.key === 'edit' the same? 
I think we're trying to deprecate lodash imports and seems a bit heavy handed to import a whole library here. ES6 Array also has includes() function that I think does the same thing.

Reply:
Yup sounds good let me change that to just filter out the key and decommission the lodash